thinkhazard/static/less/common.less

```diff
@@ -35,10 +35,16 @@ ul.horizontal li {
 }
 
 .navbar {
+  display: block;
   border: 0;
   .box-shadow(0px 2px 10px -1px rgba(0, 0, 0, 0.33));
 }
 
+.divider-vertical {
+  border-left: 1px solid #d9d9d9;
+  margin: 29px 0px;
```

Comment:
I'd prefer the vertical padding to be computed so that we can change the `@navbar-height` anytime.

If you take the `.navbar>li>a` as an example, the vertical padding take `@navbar-padding-vertical`.
https://github.com/twbs/bootstrap/blob/master/less/navbar.less#L273

This is transform to `29.5px` after less build.